test/new-e2e/tests/process/fargate_test.go

```diff
@@ -73,14 +73,27 @@ func (s *ECSFargateSuite) TestProcessCheck() {
 	assertContainersCollected(t, payloads, []string{"stress-ng"})
 }
 
-func (s *ECSFargateSuite) TestProcessCheckInCoreAgent() {
-	t := s.T()
+type ECSFargateCoreAgentSuite struct {
+	e2e.BaseSuite[environments.ECS]
+}
+
+func TestECSFargateCoreAgentTestSuite(t *testing.T) {
+	t.Parallel()
+	s := ECSFargateSuite{}
```

Comment:
```suggestion
	s := ECSFargateCoreAgentSuite{}
```

Right?

Reply:
But also your test output made me think—the two test functions are actually exactly the same right, except they are run from different test suites which have different provisioners. Is there a way to reduce the code duplication? (can be follow-up PR since we potentially can do this in three different e2e test files)

Reply:
Seems to be a common mistake :laughing: Fixed.  The best opportunity seems to be creating a function for getting payloads from fake intake. Should be able to do that quickly in another PR.